pom.xml

```diff
@@ -9,8 +9,9 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <helm-maven-plugin.version>2.4.1</helm-maven-plugin.version>
-        <helm.repository.url>https://kubernetes-charts.storage.googleapis.com</helm.repository.url>
+        <helm-maven-plugin.version>2.10.0</helm-maven-plugin.version>
+        <helm.repository.url>https://charts.helm.sh/stable</helm.repository.url>
```

Comment:
I don't think we need this at all, right? The chart has no dependencies to get and isn't published there anyway.
```suggestion
```

Reply:
yup, you're right